code/modules/vtr13/connections/blood_bonds/can_be_blood_bonded_by.dm

```diff
@@ -0,0 +1,22 @@
+/mob/living/proc/can_be_blood_bonded_by(mob/living/carbon/human/domitor)
+
+	if(!ckey || !mind)
+		return FALSE
+
+	if(!domitor || !domitor.mind)
+		return FALSE
+
+	if(!iskindred(domitor))
+		return FALSE
+
+	if(domitor.clane.name == "Tremere")
+		return FALSE
+
+	//is the thrall already in a level three blood bond?
+	if(check_character_level_three_blood_bonds(domitor))
```

Comment:
Should the arguement here be `domitor`?

Reply:
This is a wierd catch in that - yes it should, but the thing I'm checking with it, I don't actually need to check